Formula/ruby-build.rb

```diff
@@ -16,6 +16,16 @@ def install
     system "./install.sh"
   end
 
+  def caveats; <<~EOS
+    Warning: without further action ruby-build will download and install OpenSSL outside of Homebrew, and never update it.
+
+    To avoid this, set the following environment variable in your shell:
+      export RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl)"
```

Comment:
Should we specify OpenSSL 1.1 here? This is to guard ourselves from future potential `openssl@1.2` which might be incompatible with current Ruby versions
```suggestion
      export RUBY_CONFIGURE_OPTS="--with-openssl-dir=$(brew --prefix openssl@1.1)"
```